nextcord/types/components.py

```diff
@@ -69,6 +70,43 @@ class SelectMenu(TypedDict):
     disabled: NotRequired[bool]
 
 
+class UserSelectMenu(TypedDict):
```

Comment:
Same for these types in terms of inheritance, not sure how TypedDict's handle it but I assume it'd be fine?

Reply:
oliver told me to do that xD

Reply:
I was on about having a base select menu, then subclasses for the specifics

Reply:
okay, done. would this be fine?

Reply:
https://github.com/nextcord/nextcord/actions/runs/3297853193/jobs/5439199597

Basically not possible, I'll revert this change

Reply:
resolved, did the change